.travis.yml

```diff
@@ -21,7 +21,6 @@ env:
   matrix:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
-    - EMBER_TRY_SCENARIO=ember-lts-2.8
```

Comment:
Discussed with Stef RE: why this is removed. This was not supported by ember-engines.